### PR TITLE
[MOBILE-1900] Release 12.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Updates latest SDKs and fixes conflict with play services on newest Android Cordova version. I tested both iOS and Android.

### CODE REVIEW

**Review by oristanovic — APPROVED** [review]

**Comment by ralepinski** [comment]
The added google-services.json file is a mock one from https://github.com/firebase/quickstart-android/blob/master/mock-google-services.json

File: scripts/create_sample.sh [hunk 1]
```diff
@@ -42,10 +42,12 @@ cp "$ROOT_PATH/Example/index.html" www/index.html
 cp "$ROOT_PATH/Example/css/"* www/css
 cp "$ROOT_PATH/Example/js/"* www/js
 
+# copy mock google-servicces.json
```
Comment (Ulrico972) [inline scripts/create_sample.sh:45]:
```suggestion
# copy mock google-services.json
```

File: src/android/PluginManager.java [hunk 2]
```diff
@@ -325,7 +325,8 @@ public AirshipConfigOptions getAirshipConfig() {
                 .setDevelopmentLogLevel(ConfigUtils.parseLogLevel(getConfigString(DEVELOPMENT_LOG_LEVEL, ""), Log.DEBUG))
                 .setProductionLogLevel(ConfigUtils.parseLogLevel(getConfigString(PRODUCTION_LOG_LEVEL, ""), Log.ERROR))
                 .setSite(ConfigUtils.parseCloudSite(getConfigValue(CLOUD_SITE)))
-                .setDataCollectionOptInEnabled(getConfigBoolean(DATA_COLLECTION_OPT_IN_ENABLED, false));
+                .setDataCollectionOptInEnabled(getConfigBoolean(DATA_COLLECTION_OPT_IN_ENABLED, false))
+                .setUrlAllowListScopeOpenUrl(new String[]{"*"});
```
Comment (Ulrico972) [inline src/android/PluginManager.java:329]: Do we really want to initialize this config with all possible urls ? And if so, why not the other url allow lists ?

File: src/ios/UACordovaPluginManager.m [hunk 3]
```diff
@@ -136,6 +136,7 @@ - (UAConfig *)createAirshipConfig {
     airshipConfig.productionAppSecret = [self configValueForKey:ProductionAppSecretConfigKey];
     airshipConfig.developmentAppKey = [self configValueForKey:DevelopmentAppKeyConfigKey];
     airshipConfig.developmentAppSecret = [self configValueForKey:DevelopmentAppSecretConfigKey];
+    airshipConfig.URLAllowListScopeOpenURL = @[@"*"];
```
Comment (Ulrico972) [inline src/ios/UACordovaPluginManager.m:139]: Same here, why not on the other url allow lists ?

**Review by Ulrico972 — COMMENTED** [review]
Just some questions

File: src/android/PluginManager.java (hunk 2 above)
Comment (rlepinski) [reply src/android/PluginManager.java:329]: I just want to match current behavior. We can introduce behavior change in the next release

**Review by marc-scig — APPROVED** [review]
👍 with @Ulrico972's suggestion

**Pull request merged**